### DISCUSSION TITLE
UIPageControl inside TabView

### DISCUSSION THREAD

**Discussion by alinless** [siteline/SwiftUI-Introspect#296] [Ideas]
How to find a UIPageControl inside a TabView with SwiftUIIntrospect library? Now it is only UICollectionView available.
Thanks

**Comment by davdroman** [comment]
@alinless thanks for raising this. I'll be looking into it shortly.

**Comment by davdroman** [comment]
@alinless this is now implemented in master and will be part of the next release (0.9.0). Closing.

**Reply by alinless** [reply]
Thank you!
